src/html/get-metadata.js

```diff
@@ -144,7 +144,8 @@ function fallback(section) {
 }
 
 function getmetadata({ content }, { logger }) {
-  const { sections } = content;
+  const { mdast: { children } } = content;
+  const sections = children.filter(node => node.type === 'root');
```

Comment:
Can this work? Below, you are changing the type to `section`.

Reply:
👍 Again, some left-overs from playing around… though it should have failed somehow. Will double-check